public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_resource_graph/azure_resource_graph_datasource.ts

```diff
@@ -0,0 +1,233 @@
+import _ from 'lodash';
+import {
+  AzureDataSourceJsonData,
+  AzureSubscription as IAzureSubscription,
+  AzureResourceGraphQuery as IAzureResourceGraphQuery,
+} from '../types';
+import { DataSourceInstanceSettings } from '@grafana/ui';
+import { BackendSrv } from 'app/core/services/backend_srv';
+import { TemplateSrv } from 'app/features/templating/template_srv';
+import { IQService } from 'angular';
```

Comment:
We've removed the IQService from these so this should come out in the constructor and `this.$q` replace with `Promise` where ex:`return Promise.all(promises).then(results => {`